cf_units/tests/test_unit.py

```diff
@@ -373,8 +373,13 @@ def test_multiply_not_numeric(self):
     def test_multiply_with_unknown_unit(self):
         u = Unit('unknown')
         v = Unit('meters')
+        self.assertIsNotNone
         self.assertTrue((u * v).is_unknown())
         self.assertTrue((v * u).is_unknown())
+        self.assertIsNotNone((u * v).origin)
+        self.assertIsNotNone((v * u).origin)
```

Comment:
@pdearnshaw You could be stricter here and check for `_UNKNOWN_UNIT_STRING`

Reply:
I have changed this to assertEqual(..., unit._UNKNOWN_UNIT_STRING) as recommended